sql/core/src/test/resources/sql-tests/results/columnresolution-negative.sql.out

```diff
@@ -81,7 +81,7 @@ SELECT t1.i1 FROM t1, mydb1.t1
 struct<>
 -- !query 9 output
 org.apache.spark.sql.AnalysisException
-Reference 't1.i1' is ambiguous, could be: mydb1.t1.i1, mydb1.t1.i1.; line 1 pos 7
+Reference '`t1`.`i1`' is ambiguous, could be: mydb1.t1.i1, mydb1.t1.i1.; line 1 pos 7
```

Comment:
why is the new format better? it's more verbose, isn't it?

Reply:
I thought some examples in [the above](https://github.com/apache/spark/pull/22788#pullrequestreview-166797320).

Reply:
These examples only make sense when we have the outer backticks. e.g. `'t1.i1'` is good.